.rubocop.yml

```diff
@@ -5,52 +5,3 @@ inherit_gem:
 inherit_mode:
   merge:
     - Exclude
-
-AllCops:
-  TargetRubyVersion: 2.7.2
-  Exclude:
-    - 'db/**/*'
-    - 'bin/**/*'
-    - 'node_modules/**/*'
-    - 'tmp/**/*'
-    - 'vendor/**/*'
```

Comment:
I wonder why we were excluding these before?

Reply:
because we do not want to lint code that is not ours. The same rule is in the defaults I believe